lambda/translmode.ml

```diff
@@ -46,4 +46,10 @@ let transl_modify_mode locality =
 let transl_unique_barrier barrier =
   match Typedtree.Unique_barrier.resolve barrier with
   | Uniqueness.Const.Aliased -> May_be_pushed_down
-  | Uniqueness.Const.Unique -> Must_stay_here
+  | Uniqueness.Const.Unique ->
```

Comment:
This is what we agreed when talking. But now looking at the code, I wonder: is it ever possible to have `Unique` with the extension off? That would make the statement about the uniqueness extension even stronger.

Reply:
Yes, we could probably return `assert false` here